src/lib/components/index/github-card.svelte

```diff
@@ -26,7 +26,7 @@
 
 <style lang="postcss">
   .card:hover::before {
-    @apply absolute -z-10 h-[500px] w-[100px] bg-[#24292f];
+    @apply absolute -z-10 h-[500px] w-[100px] bg-[#ffffff];
```

Comment:
I think we need to differentiate between light and dark mode. Now it's working for dark mode, but in light mode, it's nearly not visible

Reply:
I will fix it..thanks tor point it out